### PR TITLE
misc

### PR DESCRIPTION
**Pull request by lnussel**

### CODE REVIEW

File: docs/Installing.asciidoc [hunk 1]
```diff
@@ -356,3 +356,35 @@ KVM doesn't work
 * make sure the 'geekotest' user can access +/dev/kvm+
 * make sure you are not already running other hypervisors such as VirtualBox
 * when running inside a vm make sure nested virtualization is enabled (pass nested=1 to your kvm module)
+
+Tests try to access openqa via 'localhost' inside qemu which fails
+==================================================================
+
+You can use the environement variable +OPENQA_HOST+ from your perl test
+scripts, so they can communicate with the real host. This variable will
```
Comment (coolo) [inline docs/Installing.asciidoc]: "*the* real host" is kind of outdated with remote workers. And the information looks wrong too - as the only test using OPENQA_HOSTNAME is yast2_lan passing it to getent ahosts - which doesn't take http urls I assume

File: script/client [hunk 2]
```diff
@@ -75,7 +76,7 @@ sub usage($) {
     }
 }
 
-GetOptions(\%options,"host=s","apibase=s","verbose|v","apikey:s","apisecret:s","help|?",) or usage(1);
+GetOptions(\%options,"host=s","apibase=s","verbose|v","apikey:s","apisecret:s", "params=s", "help|?",) or usage(1);
```
Comment (coolo) [inline script/client]: shouldn't the --help (btw, -h is documented as alias for --help in the --help output, but gives: Option h is ambiguous (help, host)) give an hint ?

File: docs/Installing.asciidoc (hunk 1 above)
Comment (lnussel) [inline docs/Installing.asciidoc]: well, if the information is wrong we should remove it then. I just moved it from Pitfalls.asciidoc

**Comment by coolo** [comment]
I fixed both complains and pushed it

**Pull request closed without merging**